superset/assets/src/SqlLab/constants.js

```diff
@@ -48,3 +48,13 @@ export const TIME_OPTIONS = [
 export const SQL_EDITOR_GUTTER_HEIGHT = 5;
 export const SQL_EDITOR_GUTTER_MARGIN = 3;
 export const SQL_TOOLBAR_HEIGHT = 51;
+
+// kilobyte storage
+export const KB_STORAGE = 1024;
+export const BYTES_PER_CHAR = 2;
+
+// browser's localStorage max usage constants
+export const LOCALSTORAGE_MAX_QUERY_AGE_MS = 24 * 60 * 60 * 1000; // 24 hours
+export const LOCALSTORAGE_MAX_USAGE_KB = 5 * 1024; // 5M
```

Comment:
I assume this is the same for all browsers?

Reply:
yes. Superset doesn't have much browser-specific code. Most features we only target Chrome.

Reply:
As a project I think we should target "modern browsers", whatever that means today and tomorrow.